package.json

```diff
@@ -88,5 +90,16 @@
   },
   "files": [
     "dist"
-  ]
+  ],
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "npm test"
+    }
+  },
```

Comment:
Personally. I prefer to have no configuration in the `package.json` file. It just adds complexity to understanding it. Let's either refactor this out into a different `.huskyrc.js` or `husky.config.js` file. I don't have a preference as to which we use. But also using a JS file will allow for comments, which might be helpful in the future.